src/AzureRMTools.ts

```diff
@@ -22,8 +23,10 @@ import * as Json from "./JSON";
 import * as language from "./Language";
 import { reloadSchemas } from "./languageclient/reloadSchemas";
 import { startArmLanguageServer, stopArmLanguageServer } from "./languageclient/startArmLanguageServer";
+import { considerQueryingForParameterFile, findMappedParamFileForTemplate, getFriendlyPathToParamFile, openParameterFile, selectParameterFile } from "./parameterFiles";
```

Comment:
>Parameter [](start = 128, length = 9)

Nit: A bit confusing that we use both "param" and "parameter". I'd stick with one or the other so it doesn't seem like there are some semantic differences (unless there are?)

Reply:
Believe I got all functions, types etc, but didn't worry about local vars.